public/views/html/shopping.html

```diff
@@ -6,20 +6,20 @@
 		<div class="container-fluid">
 			<!--an individal item, whole thing repeated for each item added -->
 			<div class="row">
-				<div ng-repeat="item in uncheckedItems" class="row topurchase">
+				<div ng-repeat="item in uncheckedItems" class="row">
 					<div class="col-xs-2 emptycb itemrow">
 						<!-- checkbox -->
 						<input type="image" src="images/icons/unchecked.png" class="checkbox" ng-click="toggleCheck(item, true)"/>
 					</div>
-					<div class="col-xs-8 itemname itemrow">
+					<div class="col-xs-8 itemrow">
 						<!-- item name injected here-->
 						<p class="shopping-item">{{item.name}}</p>
-						<!-- Tempo waste bar -->
-						<div class="col-xs-12 colourbar">
+						<!-- Waste bar -->
+						<div class="col-xs-12 colourbar" ng-style="getBackColor(item.average)">
 							<div class="innerbar" ng-style="getBarColor(item.average)">
 							</div>
+							<p>{{item.average | number: 0}}%</p>
```

Comment:
I'm personally not a huge fan of seeing the percentage right on the waste bar, but we should discuss this as a group.